### ISSUE TITLE
[no-duplicate-ng-content] new rule proposal

### ISSUE THREAD

**Issue by m-thalmann** [angular-eslint/angular-eslint#2872]
**Description and reproduction of the issue**

When using content projection in angular with `ng-content` it is not possible to reuse the same select (or default select) for multiple projections of the same content (even if it is in two different branches of an if-condition).
Therefore an error should be shown when a user tries to do this to prevent issues with displaying (angular does not warn the user about this and it just doesnt work correctly)

See: https://github.com/angular/angular/issues/9173#issuecomment-294577286

Example:

```html
@if (myCondition()) {
  <div class="a">
    <ng-content />
  </div>
} @else {
  <div class="b">
    <ng-content />
  </div>
}
```

or

```html
@if (myCondition()) {
  <div class="a">
    <ng-content select="[content]" />
  </div>
} @else {
  <div class="b">
    <ng-content select="[content]" />
  </div>
}
```

In both instances only the first ng-content will display the content, the second one will remain empty. Also this is not only an issue within conditionals also generally.

**Comment by Harpush** [comment]
I think a better rule is to flag ng-content inside control flow or templates. ng-content must be static

**Comment by m-thalmann** [comment]
@Harpush but what about the scenario when you have them displayed differently using css with a media query?
I can see multiple cases where one would use the same ng-content outside of a control flow block...
But I do agree that this is also a valid concern!

**Comment by Harpush** [comment]
Hmm you are right. I guess that won't work either. Guess those are two different rules then?
Allow single ng-content and disallow non static ng content
